src/main/java/io/netty/buffer/api/memseg/MemSegBuf.java

```diff
@@ -58,22 +61,25 @@
 
     private final AllocatorControl alloc;
     private final boolean isSendable;
+    private final int baseOffset; // TODO remove this when JDK bug is fixed (slices of heap buffers)
```

Comment:
can we report this bug via the mailing list and add the link here ?

Reply:
It's already reported. Can add a mailing list link.